### ISSUE TITLE
JSON formatter output multiple array for multiple files since v2.0.0

### ISSUE THREAD

**Issue by ybiquitous** [hadolint/hadolint#588]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Given multiple files (`Dockerfile_1` and `Dockerfile_2`), 

hadolint 1.23.0 outputs **single** array:

```console
$ docker run --rm -v ${PWD}:/work -w /work hadolint/hadolint:v1.23.0-debian hadolint --format=json Dockerfile_1 Dockerfile_2
[{"line":1,"code":"DL3006","message":"Always tag the version of an image explicitly","column":1,"file":"Dockerfile_1","level":"warning"},{"line":1,"code":"DL3006","message":"Always tag the version of an image explicitly","column":1,"file":"Dockerfile_2","level":"warning"}]
```

### Actual behavior

hadolint 2.0.0 outputs **multiple** array (an array per file):

```console
$ docker run --rm -v ${PWD}:/work -w /work hadolint/hadolint:v2.0.0-debian hadolint --format=json Dockerfile_1 Dockerfile_2
[{"line":1,"code":"DL3006","message":"Always tag the version of an image explicitly","column":1,"file":"Dockerfile_1","level":"warning"}]
[{"line":1,"code":"DL3006","message":"Always tag the version of an image explicitly","column":1,"file":"Dockerfile_2","level":"warning"}]
```

### Steps to reproduce the behavior

**Output of `hadolint --version`:**

```console
$ hadolint --version
Haskell Dockerfile Linter v2.0.0-0-gdc61382
```

**Dockerfile** (if relevant)

```Dockerfile
FROM node
```

**Additional environment details (OS, stack version, etc.)**

- OS: `Mac OS X 10.15.7 (19H15)`
- Docker: `Docker version 20.10.5, build 55c4c88`
- Docker image: `hadolint/hadolint:v2.0.0-debian`

**Comment by lorenzo** [comment]
hmm, that was unintentional, I'll take a look at it

**Comment by ybiquitous** [comment]
@lorenzo Thank you so much for the quick fix! 😄

**Comment by lorenzo** [comment]
@ybiquitous you’re welcome. I’m waiting for the GitHub actions outage to pass before I can publish the new version

**Comment by ybiquitous** [comment]
@lorenzo Thanks for the response. I'm not in hurry, so no worries. 👍
